tutorials/using-bucket-website-with-mkdocs/index.mdx

```diff
@@ -0,0 +1,380 @@
+              example: octicons/beaker-16
+              quote: octicons/quote-16
+        ```
+
+    #### Using admonitions
+
+        The example below shows you the syntax you need to use in your files for admonitions. You can replace `info` with any of the admonitions you have added in the previous section (`abstract`, `note`, `tip`, `success`, `question`, `warning`, `failure`, `danger`, `bug`, `example`, and `quote`).
+
+        ```
+        !!! info
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        You can add a custom title inside the admonition box like in the example below.
+
+        ```
+        !!! info "Information about the admonition box"
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        If you want your admonition box to be collapsible, you can use the code below. The user will only see the box without its content.
+
+        ```
+        ??? info
+
+            I am an information admonition. You can use me to give additional information to your user.
+        ```
+
+        Find out more about admonitions on the [MkDocs website](https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=admonitions#configuration).
+
+    ### Additional configuration
+
+        You can add the following additional configuration options in your `mkdocs.yml` file. The `features` component allows you to use various features of the Material theme.
+          ```
+          features:
+            - content.code.copy
+            # Adds a copy button to code blocks.
+            - navigation.sections
+            # Enables section-based navigation.
+            - navigation.path
+            # Shows the navigation path (breadcrumbs).
+            - navigation.top
+            # Adds a "back to top" button.
+            - search.suggest
+            # Enables search suggestions.
+          ```
+        Add the `plugins` component to define the MkDocs plugins you want to enable on your website. The `search` plugin allows you to add the search functionality and the `glightbox` plugin allows you to add images and videos.
+          ```
+          plugins:
+            # Lists the MkDocs plugins enabled for the site.
+            - search
+            # Adds search functionality.
+            - glightbox
+            # Adds lightbox functionality for images and videos.
+          ```
+        Add the copyright notice for your website.
+          ```
+          copyright: >
+            Copyright &copy; 2016 - 2024 Martin Donath
+          ```
+    ## Downloading the mkdocs file template
+
+        If you have only added the elements shown in the sections above, you can download the `mkdcos.yml` below.
+
+          <a href="/tutorials/using-bucket-website-with-mkdocs/mkdocs-example.yml" target="_blank" rel="noopener noreferrer" download="mkdocs-example.yml"> Download the mkdocs file </a>
```

Comment:
There is a probability that this won't work for security reasons, I would suggest adding a section "## Example MkDocs configuration file" with the content of the mkdocs-example.yml" file

Reply:
Or either put the file into an Object Storage bucket, but I prefer Samy's suggestion

Reply:
Removed this part